jdaviz/core/astrowidgets_api.py

```diff
@@ -166,7 +166,11 @@ def offset_by(self, dx, dy):
 
     @property
     def zoom_level(self):
-        """Zoom level:
+        """
+        The zoom level for an image viewer (not linked by WCS).
+
+        Warning: when a viewer is linked by WCS, the result corresponds

```

Comment:
Make it more obvious.
```suggestion
        .. warning:: when a viewer is linked by WCS, the result corresponds
```